src/containers/Chat/ChatConversations/ConversationList/ConversationList.tsx

```diff
@@ -121,7 +121,7 @@ export const ConversationList: React.SFC<ConversationListProps> = (props) => {
           lastMessage={lastMessage}
           senderLastMessage={conversation.contact.lastMessageAt}
           contactStatus={conversation.contact.status}
-          contactProviderStatus={conversation.contact.providerStatus}
+          contactProviderStatus={conversation.contact.bspStatus}
```

Comment:
We should also change contactProviderStatus to "contactBspStatus" and then the respective components